src/main/java/mods/railcraft/common/blocks/wayobjects/RoutingLogic.java

```diff
@@ -204,8 +222,8 @@ public NOT(Condition a) {
         }
 
         @Override
-        public boolean matches(IRoutingTile tile, EntityMinecart cart) {
-            return !a.matches(tile, cart);
+        public int evaluate(IRoutingTile tile, EntityMinecart cart) {
+            return valueForBool(a.evaluate(tile, cart) == 0);
```

Comment:
That's a different behavior, though worth considering. We'd have to nix the special -1 though. My original thought was that most devices treat any power level as "on", so the inverse should be "off" (i.e. `== 0` behaves like a redstone torch).